schema_based_on_diagram.sql

```diff
@@ -0,0 +1,51 @@
+-- Creating patients table
+CREATE TABLE patients (
+  id            SERIAL PRIMARY KEY,
+  name          VARCHAR(255),
+  date_of_birth DATE
+);
+
+-- Creating medical_histories table
+CREATE TABLE medical_histories (
+  id         SERIAL PRIMARY KEY,
+  admited_at TIMESTAMP,
+  patient_id INTEGER UNIQUE REFERENCES patients(id),
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) all FOREIGN KEYS should have indexes created. for example  `CREATE INDEX ON medical_histories (patient_id);`